src/styles/styles.js

```diff
@@ -290,18 +290,16 @@ const styles = {
 
     picker: {
         inputIOS: {
-            fontFamily: fontFamily.GTA,
-            fontSize: variables.fontSizeNormal,
-            paddingLeft: 12,
-            paddingRight: 12,
-            paddingTop: 10,
-            paddingBottom: 10,
-            borderRadius: variables.componentBorderRadius,
+            flex: 1,
             borderWidth: 1,
+            borderRadius: variables.componentBorderRadiusNormal,
             borderColor: themeColors.border,
-            color: themeColors.text,
-            height: variables.inputComponentSizeNormal,
-            opacity: 1,
+            paddingTop: 25,
```

Comment:
Just noting here that prettymuch all of our padding and spacing throughout the app comes in increments of 4. So I feel like this should be `24` instead of `25`, but cc @shawnborton for confirmation.